sonar/theme/assets/scss/sonar/theme.scss

```diff
@@ -1,16 +1,3 @@
-/*
```

Comment:
please add your own licence

Reply:
I erased license by error.. I noticed that several files don't have license, I will do a PR for that and for commenting files, too.